README.md

```diff
@@ -1,4 +1,4 @@
-# An OOP school library app: Decorator pattern in OOP 🎁
+# An OOP school library app: Set up associations 🏹
```

Comment:
[OPITONAL] Make sure next time you added the MIT license correctly. When I tried to open it in Readme it shows me the broken link.